Source/Extensions/Blazorise.Video/VideoJSOptions.cs

```diff
@@ -2,82 +2,84 @@
 
 namespace Blazorise.Video;
 
-internal class VideoJSOptions
+public class VideoInitializeJSOptions
 {
-    [JsonPropertyName( "controls" )]
     public bool Controls { get; set; }
 
-    [JsonPropertyName( "controlsDelay" )]
     public double ControlsDelay { get; set; }
 
-    [JsonPropertyName( "controlsList" )]
     public string[] ControlsList { get; set; }
 
-    [JsonPropertyName( "settingsList" )]
     public VideoSettingsType[] SettingsList { get; set; }
 
-    [JsonPropertyName( "automaticallyHideControls" )]
     public bool AutomaticallyHideControls { get; set; }
 
-    [JsonPropertyName( "autoplay" )]
```

Comment:
Be careful about the casing. `autoplay` vs `autoPlay`. Make sure everything matches what is used in JS.